lang/en.json

```diff
@@ -285,6 +295,21 @@
       "header": "Accessories",
       "empty": "Equip up to {slots} accessories from your Backpack toggling the shield icon."
     },
+    "ACTIONS_ATTACK": {
+      "header": "Attack Actions"
+    },
+    "ACTIONS_GENERAL": {
+      "header": "General Actions"
+    },
+    "ACTIONS_MOVEMENT": {
+      "header": "Movement Actions"
+    },
+    "ACTIONS_REACTION": {
+      "header": "Reactions"
+    },
+    "ACTIONS_SPELL": {
+      "header": "Spellcraft Actions"
+    },
```

Comment:
I don't love a different namespace for these, maybe we can group everything under something like `ACTOR.SHEET.HEADERS`?